schemas/code_signature.yml

```diff
@@ -0,0 +1,59 @@
+---
+- name: code_signature
+  title: Code Signature
+  group: 2
+  description: These fields contain information about binary code signatures.
+  type: group
+  reusable:
+    top_level: false
+    expected:
+      - file
+      - process
+      # - dll
+      # - driver
+  fields:
+
+    - name: subject_name
+      format: string
+      level: core
+      type: keyword
+      description: Subject name of the code signer
+      example: Microsoft Corporation
+
+
+    - name: valid
+      level: core
+      type: boolean
+      description: Boolean to capture if the digital signature is verified against the binary content.
+      example: "true"
+
+
+    - name: exists
+      format: core
+      level: extended
+      type: keyword
+      short: Boolean to capture if a signature is present.
+      description: >
+        Boolean to capture if a signature is present.
+
+        This should only populated if the signature was checked.
```

Comment:
> This should only populated if the signature was checked. 

Why? Isn't this overloading the definition of `code_signature.exists`? Whether a signature exists is orthogonal to whether its validity has been assessed, IMO.

The ternary approach would make more sense on `code_signature.valid`, though:

* haven't checked validity: `code_signature.valid` is absent
* checked validity, good: `code_signature.valid=true`
* checked validity, bad: `code_signature.valid=false`

Or am I missing something?

If you agree, we should adjust the definitions of `valid` and `exists` accordingly.